src/utils/stack.ts

```diff
@@ -40,6 +41,20 @@ export class StackBuilder {
     );
     this.code.line("super(scope, id, props);");
 
+    if (this.config.yamlTemplateLocation) {
+      this.code.line(
+        `const yamlTemplateFilePath = join(__dirname, "../..", "${this.config.yamlTemplateLocation}");`
```

Comment:
This code is for the generated file on path `repo-root/cdk/lib/my-app.ts`. `__dirname` is the directory location of the TS file, so we locate the YAML file relative to `repo-root`.